src/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.csproj

```diff
@@ -5,11 +5,17 @@
     <TargetFrameworks Condition="'$(UseIntrinsics)' == 'true'">netstandard2.0;netcoreapp3.0</TargetFrameworks>
     <IncludeInPackage>Microsoft.ML.CpuMath</IncludeInPackage>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DefineConstants>$(DefineConstants);PRIVATE_CONTRACTS</DefineConstants>
```

Comment:
Is it odd to use the `PRIVATE_CONTRACTS` define to now include both `BestFriendAttribute` and `PublicKey`? Should we come up with a better name for this define?  How about `USE_HIDDEN_INFRASTRUCTURE` or something similar?

Reply:
That flag is renamed to CPUMATH_INFRASTRUCTURE because they are used to make CPUMATH a standalone module.

---
In reply to: [236739910](https://github.com/dotnet/machinelearning/pull/1724#discussion_r236739910) [](ancestors = 236739910)